src/containers/Governance/GovernanceOverview/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {FC} from 'react';
+
+import {DocContainer, DocImage} from 'components';
+
+import Overview from './GovernanceOverview.png';
+
+const GovernanceOverview: FC = () => {
+  return (
+    <DocContainer className="GovernanceOverview" title="Overview" lastUpdated="15 Feb 2021">
+      <DocImage alt="governance-overview" maxWidth={1200} src={Overview} />
```

Comment:
@gsonly Can you also add a glossary after the overview image? 

Like this: https://thenewboston.com/guide/introduction

Reply:
Done